src/views/Give/Give.tsx

```diff
@@ -96,14 +99,20 @@ function Give({ selectedIndex }: GiveProps) {
               </Tabs>
 
               <TabPanel value={view} index={0}>
-                <CausesDashboard />
+                <GohmToggle giveAssetType={giveAssetType} changeAssetType={changeAssetType} />
```

Comment:
The alignment on this with the CausesDashboard container is slightly off and I can't figure out how to fix it without adding custom css...